src/ispc/kernels/lanczos3.ispc

```diff
@@ -63,10 +55,14 @@ static inline uint8<4> resample_internal(uniform Image src_image, uniform float<
             float wy = lanczos3_filter((uniform float)y + lanczos3_offset);
             float w = wx * wy;
             float<2> texel_offset = {x, y};
-            float<2> texel_uv = uv + texel_offset * inv_src_size;
-            int<2> pixel_coord = uv_to_pixel_id(src_image.size, texel_uv);
+            float<2> src_uv = uv * src_image.size + texel_offset;
+            int<2> src_coord = src_uv + 0.5; // TODO: uv is already centered, this offset seems wrong?
```

Comment:
Maybe make the conversion explicit? And I'm also of the opinion that this is wrong/not necessary. We already have the center pixel of our kernel calculated in the line above, there shouldn't be a need for any offsets at that point